presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -616,7 +618,11 @@ private void testCreatePartitionedBucketedTableAsFewRows(Session session, HiveSt
             fail("expected failure");
         }
         catch (Exception e) {
-            assertEquals(e.getMessage(), "Cannot insert into existing partition of bucketed Hive table: partition_key=c");
```

Comment:
While we are refactoring this, update to `assertThatThrownBy()`

http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion